spacy_llm/ty.py

```diff
@@ -17,7 +17,7 @@
 
 PromptExecutor = Callable[[Iterable[_Prompt]], Iterable[_Response]]
 ExamplesConfigType = Union[
```

Comment:
This can still be None for other tasks (and actually I think we could maybe still allow it to None for this COT version of the NER task)